distributed/client.py

```diff
@@ -1266,7 +1266,7 @@ def _handle_error(self, exception=None):
                 self._release_key(key=key)
             if self._start_arg is None:
                 with ignoring(AttributeError):
-                    await self.cluster._close()
+                    await self.cluster.close()
```

Comment:
This is only called for `LocalCluster`, correct?

Reply:
I think it's any cluster that is given to the client in a situation like the following:

```python
cluster = FooCluster()
client = Client(cluster)
```

Generally, how do you feel about agreeing on a `close` method?

Reply:
if `self._start_arg is None` makes me think it's just `client = Client()`. IMO we shouldn't have the closure of a client shutdown a remote cluster, the cluster object can manage that itself just fine.

I'm fine with agreeing on a close method, I believe we already standardize on this.

Reply:
Oh indeed, you're correct about `_start_arg is None` and `Client()`.